app/models/user.rb

```diff
@@ -37,6 +37,11 @@ def self.new_with_session(params, session)
     end
   end
 
+  def self.search(input, search)
+    input.where("firstname LIKE ?", "%#{search}%")
+  end
```

Comment:
Oh.. I think this may be meant to be a separate class (service class?!) cause it's designed to search multiple models. I'm actually not sure if User.where would work when the method is already in the user model. But then the column name needs to be changeable too? Not sure haha. We can look into fixing both our search functions tomorrow.